stable/aws-for-fluent-bit/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-for-fluent-bit
 description: A Helm chart to deploy aws-for-fluent-bit project
-version: 0.1.31
+version: 0.1.32
```

Comment:
Should the version be a breaking change, 0.2.0?

Reply:
In this repo, historically, changing default values is not considered a breaking change. Users can and should customize the values.yaml, the one provided is just recommended defaults. When they upgrade the chart version they keep their own values.yaml if they use the reuse-values flag: https://stackoverflow.com/questions/62249179/how-to-persist-set-key-values-to-values-yaml-in-helm-install-upgrade